.travis.yml

```diff
@@ -16,3 +16,17 @@ script:
   - python -m jupyterlab.browser_check
   - jlpm run lint
   - python tests/test-browser/run_browser_test.py
+
+deploy:
+  - provider: pypi
+    user: "Your username"
+    password:
+      secure: "Your encrypted password"
+    distributions: "sdist bdist_wheel"
+    on:
+      tags: true
+  - provider: npm
+    email: "YOUR_EMAIL_ADDRESS"
+    api_key: "YOUR_AUTH_TOKEN"
```

Comment:
Is this plain text?

Reply:
Tokens needs to be encrypted through travis cli (this is explained in the documentation). But I did not find any information on whether the username/email may be encrypted or not.